### PR TITLE
Shopify X Floship 💣

### PR DESCRIPTION
**Pull request by muhajirdev**

### CODE REVIEW

File: helper/shopify.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//   SHOPIFY_HOST,
+// } = process.env
+
+// Shopify.Context.initialize({
+//   API_KEY: SHOPIFY_API_KEY,
+//   API_SECRET_KEY: SHOPIFY_API_SECRET_KEY,
+//   SCOPES: [SHOPIFY_SCOPES],
+//   HOST: SHOPIFY_HOST.replace(/https:\/\//, ''),
+//   API_VERSION: ApiVersion.October21,
+// })
```
Comment (celest1al) [inline helper/shopify.js:18]: is it really this complex to access our shopify orders? it's simple enough from this docs https://shopify.dev/api/admin-rest/2021-10/resources/order @muhajirdev

Comment (muhajirdev) [reply helper/shopify.js:18]: yep, the REST API is simple.
They have node-sdk`@shopify/shopify-api` which is unnecassarily complex for our use case.

so we just use REST API for now. and it works well 👍

Comment (celest1al) [reply helper/shopify.js:18]: nice info gan 👍

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": ["@babel/preset-env"],
+  "targets": {
+    "node": "v12.13.0"
```
Comment (celest1al) [inline .babelrc:4]: why use node 12? @muhajirdev

Comment (muhajirdev) [reply .babelrc:4]: netlify is using node 12 by default 
image

https://docs.netlify.com/functions/build-with-javascript/#runtime-settings

**Review by celest1al — APPROVED** [review]

**Pull request merged**